.eslintrc.js

```diff
@@ -53,5 +53,12 @@ module.exports = {
         '@typescript-eslint/no-unused-expressions': 'off',
       },
     },
+    {
+      // disable propTypes checking in typescript files. many false positives with destructuring
```

Comment:
I actually can remove that because I turned off the lint rule in tsx files since the eslint plugin doesn't properly understand nontrivial typescript types.